web/src/utils/navigation.ts

```diff
@@ -23,4 +23,5 @@ export function navigateTab(dashboard: string, tab: string){
     url.pathname = `${dashboard}/${tab}`
   }
   history.pushState(null, '', url);
+  window.dispatchEvent(new CustomEvent('location-changed'));
```

Comment:
could you create another function or change this one? e.g. add a boolean to control the event dispatch behaviour?

tab bar relies on `navigateTab` to not reload, but for your case, we should.

Reply:
Done.